RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -91,7 +91,6 @@ class CConfig():
     sProjectName      = None
     iSuiteCount       = 0
     iTestCount        = 0
-    iTotalTestcases   = 0
```

Comment:
Hi Son,
thank you four your investigation about this topic.
How does the robot code look like now to access the total number of test cases?
Thank you,
 Thomas

Reply:
The total testcaes is counted by ROBFW core already.

Reply:
Hi Son,

O.K., but how can we access this.
The requirement is that we have a variable where we can calculate the progress of the test execution as part of our robot code.
Thank you,
 Thomas

Reply:
Hello Thomas,

   In config object, we have iTestcount variable, it's auto increased in each test teardown => we can track test cases executed by using this variable.
   Currently, the number of test cases executed is printed out in our ROBFW-AIO log.

Thank you,
Son

Reply:
Hi Son,
very good. Please give me an example how to access iTestcount with robot code.
Thank you,
 Thomas